src/pages/List/ListImages/ListImages.js

```diff
@@ -0,0 +1,624 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useState } from 'react';
```

Comment:
같은 패키지이기 때문에 한 줄로 import하시면 될 것 같습니다.
```suggestion
import React, { useState } from 'react';
import styled from 'styled-components';
```